akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -248,61 +248,70 @@ private[remote] class Decoder(
             OptionVal.None
         }
 
```

Comment:
it's mostly whitespace changes, see https://github.com/akka/akka/pull/21342/files?w=2